x-pack/plugins/enterprise_search/public/applications/app_search/components/data_panel/data_panel.tsx

```diff
@@ -30,6 +30,7 @@ interface Props {
   titleSize?: EuiTitleProps['size'];
   subtitle?: React.ReactNode;
   iconType?: EuiIconProps['type'];
+  icon?: React.ReactElement;
```

Comment:
I don’t think you actually need to add an icon prop to DataPanel . The type property of an EuiIcon component already accepts a ReactElement, for instance [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.tsx#L74) we’re already using that functionality to pass a custom SVG icon into a DataPanel.

Reply:
Oh nice! I had no idea, I appreciate that. I will make the change.

Reply:
613add2bdf5eb0279a90f0d02fd9e1a821542125